docs/onboarding-tech/3-network.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2024-02-12
+category: document
+author:  Samson Rinaldi / Chris Punt
+order: 3
+---
+
+# Networking: How We're Connected
+
+Welcome to a quick dive into how networking powers connectivity at Jangala! Whether you're tech-savvy or just curious about the technical underpinnings, this guide will provide you with key insights into how our products share connectivity.
+
+## Wi-Fi Connectivity
+
+Our devices broadcast Wi-Fi at two main frequencies: 2.4GHz and 5GHz, which are part of the radio band of the electromagnetic spectrum. The 2.4GHz band offers extensive coverage and greater range, making it ideal for reaching through walls and other barriers. In contrast, the 5GHz band provides faster data speeds, perfect for high-bandwidth activities like streaming and gaming.
+
+## Backhaul Options
+
+To connect to the broader internet, our devices use three types of backhaul: cable, satellite, and cellular towers. Each has its advantages — cable is reliable for steady connections, satellite covers wide areas beyond other infrastructures, and cellular offers flexibility with mobile access. This diversity ensures that our devices maintain connectivity in various settings.
+
+## IP Addresses
+
+Each device connected to our Wi-Fi is assigned an IP address, akin to a postal address on the internet. This ensures that data packets are accurately delivered to and from your device, whether it's communicating within your home network or across the internet.
+
+## Routing and Data Retrieval
+
+When you send a request from your device, it is routed through the network to access the internet. Initially, the request moves to a Wide Area Network (WAN), reaches a DNS server to translates the domain name into an IP address, and then proceeds to the appropriate data center to retrieve the requested data. The response travels back along this path to reach your device, guided by the IP address.
+
+Through understanding these fundamental elements—from Wi-Fi transmission and backhaul options to IP addresses and routing—you'll appreciate how our devices facilitate reliable and efficient internet connections. This knowledge is essential as it ensures every interaction with our technology is smooth and productive. Welcome to the interconnected world of Jangala, where advanced networking brings your digital world to life.
```

Comment:
This sounds very ChatGPT, put this in our own words

Reply:
done